ThunderTable/TableSection.swift

```diff
@@ -23,6 +23,10 @@ public protocol Section {
 	var editHandler: EditHandler? { get set }
     
     var selectionHandler: SelectionHandler? { get set }
+	
+	var rowLeadingSwipeActionsConfiguration: SwipeActionsConfigurable? { get }
```

Comment:
Indentation seems a bit off here, guessing Github has got confused with tabs and spaces again? :)